packages/ui/src/modal/bid/BidModalRenderer.tsx

```diff
@@ -8,25 +8,19 @@ import {
   useAttributes,
   useChainCurrency,
 } from '../../hooks'
-import {
-  useAccount,
-  useBalance,
-  useNetwork,
-  useSigner,
-  mainnet,
-  goerli,
-} from 'wagmi'
-
-import { constants } from 'ethers'
+import { useAccount, useBalance, useNetwork, useWalletClient } from 'wagmi'
+import { mainnet, goerli } from 'wagmi/chains'
```

Comment:
Are these chains required? Not seeing them used anywhere.

Reply:
Yeah, they're used in determining the `convertCurrencyLink`